src/containers/availableTeams/ducks/selectors.ts

```diff
@@ -0,0 +1,18 @@
+import { AsyncRequest, AsyncRequestKinds } from '../../../utils/asyncRequest';
+import { AvailableTeam } from './types';
+import { TabItem } from '../../../components/leaderboard/types';
+
+export const availableTeamsToTabItems = (
+  availableTeams: AsyncRequest<AvailableTeam[], any>,
+): TabItem[] => {
+  if (availableTeams.kind === AsyncRequestKinds.Completed) {
```

Comment:
⛏️ We might want to use AsyncRequestIsCompleted here instead of checking kind directly

Reply:
agree, utilities should be preferred